### PR TITLE
REG-430 Upgrade sandbox

### PR DESCRIPTION
**Pull request by aquiladev**

### CODE REVIEW

File: sandbox/index.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ class Sandbox {
     this.options = options || {};
     this.provider = service.provider;
     this.snapshotId = undefined;
-    this.version = 'v0.5';
+    this.version = 'v0.6';
```
Comment (bogdan) [inline sandbox/index.js:81]: it is odd to have `v` in version. I think we should avoid it:

```suggestion
    this.version = '0.6';
```

File: sandbox/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## v0.6
+- Rebuild sandbox (UNS v0.4.0)
```
Comment (bogdan) [inline sandbox/CHANGELOG.md:2]: Why is sandbox version different from UNS version? I think having two is a confusion.

Comment (aquiladev) [reply sandbox/CHANGELOG.md:2]: sandbox has a bit different livecycle (as least it had). sandbox is the part of the project which can be moved to different repo, not everybody requires this tool. I'm sure most of a repos require the **uns** only for config and abis. that is why I decided to use simplified versioning.

Comment (bogdan) [reply sandbox/CHANGELOG.md:2]: Sandbox should supply the UNS version besides its own. Maybe UNS version is even more important than sandbox version itself. Maybe we can only consider sandbox as a subversion of UNS itself, like if sandbox is based on UNS `v0.4` then sandbox version can only be `0.4.x`. What do you think?

Comment (aquiladev) [reply sandbox/CHANGELOG.md:2]: I'll align all versions as soon we finish the epic, because there are more parts in the project where version is present

**Review by bogdan — APPROVED** [review]

**Pull request merged**